### PR TITLE
add laspy

### PR DESCRIPTION
**Pull request by daavoo**

### CODE REVIEW

File: recipes/laspy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/laspy/meta.yaml]: Please delete all comments.

File: recipes/laspy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "laspy" %}
+{% set version = "1.5.0" %}
+{% set sha256 = "9561d6d4cfb5ebfa8201151415d45085" %}
```
Comment (synapticarbors) [inline recipes/laspy/meta.yaml]: I believe the sha256 hash should be:
329dc86a1296996a22d110ea8c155de8567089f5a9e5c7f02843725cc28cd3ab

File: recipes/laspy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  build:
+    - python
+    # When setuptools is available add the `--single-version-externally-managed --record record.txt` above.
+    - setuptools
+    # if your project compiles code (such as a C extension) then add `toolchain` as a build requirement.
+  run:
+    - python
+    - numpy
+
+test:
```
Comment (synapticarbors) [inline recipes/laspy/meta.yaml]: It looks like there are a number of command line scripts:

https://github.com/laspy/laspy/blob/master/setup.py#L42-L48

Please add the appropriate `command` tests:

https://conda.io/docs/building/meta-yaml.html#test-commands

File: recipes/laspy/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (synapticarbors) [inline recipes/laspy/meta.yaml]: Please add `entry_points` for all of the command line tools:

https://conda.io/docs/building/meta-yaml.html#python-entry-points

https://github.com/laspy/laspy/blob/master/setup.py#L42-L48

File: recipes/laspy/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGLP.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
+  license: BSD-3-Clause
+  # The license_family, i.e. "BSD" if license is "BSD-3-Clause". (optional)
+  license_family: BSD
+  # It is strongly encouraged to include a license file in the package,
+  # (even if the license doesn't require it) using the license_file entry.
+  # See http://conda.pydata.org/docs/building/meta-yaml.html#license-file
+  license_file: LICENSE.txt
```
Comment (synapticarbors) [inline recipes/laspy/meta.yaml]: It looks like the `MANIFEST.in` doesn't specify the license file so it doesn't show up in the source distribution on pypi and will fail. See the following as a work-around:

http://conda-forge.github.io/docs/meta.html#packaging-the-license-manually

File: recipes/laspy/meta.yaml (hunk 2 above)
Comment (daavoo) [inline recipes/laspy/meta.yaml]: Thanks for the help @synapticarbors . May I ask where did you get this sha256 hash??? I checked the md5 links on https://pypi.python.org/pypi/laspy#downloads but none of them shows that number.

Comment (synapticarbors) [inline recipes/laspy/meta.yaml]: When getting the source from pypi.io, you can get it from the page there:
https://pypi.org/project/laspy/#files

or generically, by following these instructions:

http://conda-forge.github.io/docs/meta.html#populating-the-hash-field

**Pull request closed without merging**